stdlib/universe/mode.go

```diff
@@ -0,0 +1,519 @@
+package universe
+
+import (
+	"fmt"
+
+	"github.com/influxdata/flux/values"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/execute"
+	"github.com/influxdata/flux/plan"
+	"github.com/influxdata/flux/semantic"
+)
+
+const ModeKind = "mode"
+
+type ModeOpSpec struct {
+	Column string `json:"column"`
```

Comment:
If I understand correctly, mode is supposed to be an aggregate which means it should match with the same behavior as the other aggregates. We have `execute.AggregateConfig` and other related functions in `execute/aggregate.go` that handle some of the extra portions of this scenario.

I think that pattern would be better for this scenario. Can you take a look at things like `stdlib/universe/count.go` and try to match that pattern? I also think there's a reasonable chance it would resolve some of my other comments.

Reply:
see below conversation